packages/api/src/shared/config.ts

```diff
@@ -276,4 +276,8 @@ export class Config {
   static getCxsWithEnhancedCoverageFeatureFlagName(): string {
     return getEnvVarOrFail("CXS_WITH_ENHANCED_COVERAGE_FEATURE_FLAG");
   }
+
+  static getIheARN(): string {
+    return getEnvVarOrFail("CERT_ARN");
+  }
```

Comment:
I suggest we be more specific when naming stuff, in this case this is not IHE's ARN, its IHE Cert's ARN. And its not just any cert ARN (`CERT_ARN`).

It's a good practice to look around for existing examples. For CW, this is called `getCWOrgCertificate` and `getCWMemberCertificate` - see the respective env vars for reference too.

Also, we usually have acronyms/abbreviations in all caps. So maybe rename this to `getIHECertificateArn()` - example: `getCWManagementCookieArn`

Reply:
By the way, where is this being used? No ref for `getIheARN` on this PR right?

Reply:
good call. removed entirely.